Rakefile

```diff
@@ -11,7 +11,8 @@ Vmdb::Application.load_tasks
 begin
   require 'jasmine'
   load 'jasmine/tasks/jasmine.rake'
-rescue LoadError
+rescue LoadError => e
+  puts e.message
```

Comment:
I can suppress rubocop check by # rubocop:disable HandleExceptions or by adding exclude to the .rubocop.yml(in this case it will skip HandleExceptions check for the whole file)